### DISCUSSION TITLE
Not ord related but a suggestion for the ordinals collections standards

### DISCUSSION THREAD

**Discussion by StellarStoic** [ordinals/ord#2012] [Ideas]
As you all know, the collections are usually merged with the main branch of the [ordinals-wallet repo](https://github.com/ordinals-wallet/ordinals-collections). they have this standard that is completely fine from a technical perspective but when it comes to the actual workflow of our pull requests and merging them to the main branch, it's a completely crappy process. I'm suggesting the following. [OpenOrdex](https://github.com/orenyomtov/openordex) from @orenyomtov is a true trustless marketplace and their repo should become the collections hub that will all adopt or if he disagrees we can make the ord collections standard here. The merging process could be done with [CircleCi](https://circleci.com/) scripts automatically if all checks are green. 
there should be no biased middlemen in a trustless ordinals market.

**Comment by cliobitcoinbank** [comment]
Supposedly the parent child feature will fix this so maybe thats a better complaint for you.  IMO ordinals-wallet has done a really great job maintaining the collections list and I'm glad sites like ord.io are picking up the standard.

**Reply by StellarStoic** [reply]
Also banks are doing a amazing jobs, yet they are centralized.

The collection standard have a single point of failure and that's centralization atm. I hope this will change in the future.

Try to import your key ~~ord.io~~ ordawap gave you into other wallets. 
You can't and this is a huge red flag.

**Reply by cliobitcoinbank** [reply]
The parent child feature will fix this. Relax.
